src/utils/safe-utils.ts

```diff
@@ -7,11 +7,3 @@ export function chuzzle(input: number) {
 export function panic(message?: string): never {
   throw new Error(message)
 }
-
-export function flatmap<S, T>(t: T | undefined | null, body: (t: T) => S | undefined, opts?: {rescue?: boolean}): NonNullable<S> | undefined {
```

Comment:
my beloved flatmap!?!?!

Reply:
I guess we no longer use it. So indeed removal is the right thing.

For coverage 🫡

Reply:
it's in the codebase twice actually, I removed the unused version.

Reply:
He loves it so much.

Reply:
lol. incredible